src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RefundConfirmationConfirmation.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static java.util.Objects.requireNonNull;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PostSubmitCallbackHandler;
+
+@Component
+public class RefundConfirmationConfirmation implements PostSubmitCallbackHandler<AsylumCase> {
```

Comment:
Can you add a quick functional test for this confirmation page please?

Reply:
Have added a FT for this now